front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -953,16 +955,16 @@ function AdvancedSettings({
   setMaxStepsPerRun: (maxStepsPerRun: number | null) => void;
 }) {
   return (
-    <Popover
-      trigger={
+    <PopoverRoot>
```

Comment:
Why not use the Popover component directly with popoverTriggerAsChild?

Reply:
Totally forgot with have that props.. 🙈

Reply:
+1 with @aubin-tchoi